cmd/snap/cmd_snap_op.go

```diff
@@ -665,7 +665,13 @@ func (x *cmdRefresh) showRefreshTimes() error {
 		return err
 	}
 
-	fmt.Fprintf(Stdout, "schedule: %s\n", sysinfo.Refresh.Schedule)
+	timerOrSchedulePrefix := "timer"
+	timerOrSchedule := sysinfo.Refresh.Timer
+	if sysinfo.Refresh.Timer == "" && sysinfo.Refresh.Schedule != "" {
+		timerOrSchedulePrefix = "schedule"
+		timerOrSchedule = sysinfo.Refresh.Schedule
+	}
+	fmt.Fprintf(Stdout, "%s: %s\n", timerOrSchedulePrefix, timerOrSchedule)
```

Comment:
That's quite verbose for what's essentially this:
```
if sysinfo.Refresh.Schedule != "" {
        fmt.Fprintf(Stdout, "schedule: %s", sysinfo.Refresh.Schedule
} else {
        fmt.Fprintf(Stdout, "timer: %s", sysinfo.Refresh.Timer)
}
```
Note that the server should never send a schedule if timer is set, or the reverse.